.gitlab-ci.yml

```diff
@@ -983,6 +993,7 @@ dogstatsd_rpm-x64:
   stage: package_build
   image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-buildimages/rpm_x64:$DATADOG_AGENT_BUILDIMAGES
   tags: [ "runner:main", "size:large" ]
+  needs: [ "build_dogstatsd-deb_x64" ]
```

Comment:
Seems weird that the `rpm` dogstatsd package build depends on the `deb` dogstatsd build, but since we don't have an `rpm` dogstatsd build, this is fine.